apps/vercel/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,118 @@
+import { useCallback, useState, useEffect, ChangeEvent } from 'react';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import {
+  Box,
+  Flex,
+  FormControl,
+  Heading,
+  HelpText,
+  Paragraph,
+  Stack,
+  TextInput,
+  TextLink,
+} from '@contentful/f36-components';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { styles } from '../components/config-screen/ConfigScreen.styles';
+import VercelIcon from '../components/common/VercelIcon';
+
+export interface AppInstallationParameters {
+  accessToken?: string;
+}
+
+const ConfigScreen = () => {
+  const sdk = useSDK<ConfigAppSDK>();
+  const [parameters, setParameters] = useState<AppInstallationParameters>({});
+  const [accessToken, setAccessToken] = useState<string>('');
```

Comment:
I feel like we could cut down on duplicate data & some moving parts by removing accessToken and just using parameters instead

Reply:
yep makes sense, that just for the initial controlled input honestly as we are going to implement an OAUTH flow